pyproject.toml

```diff
@@ -17,10 +17,11 @@ dependencies = [
     "numpy>=1.17",
     "pandas",
     # NOTE: Keep minimum pymc version in sync with ci.yml `OLDEST_PYMC_VERSION`
-    "pymc>=5.3.0",
+    "pymc>=5.5.0",
```

Comment:
You need to update `OLDEST_PYMC_VERSION` in `ci.yml` as mentioned in the comment above

Reply:
updated

Reply:
updated